test/e2e/common/downward_api.go

```diff
@@ -78,7 +78,7 @@ var _ = Describe("[sig-node] Downward API", func() {
 		expectations := []string{
 			fmt.Sprintf("POD_NAME=%v", podName),
 			fmt.Sprintf("POD_NAMESPACE=%v", f.Namespace.Name),
-			"POD_IP=(?:\\d+)\\.(?:\\d+)\\.(?:\\d+)\\.(?:\\d+)",
+			"POD_IP=(?:\\d+\\.\\d+\\.\\d+\\.\\d+|[a-fA-F0-9:]+)",
```

Comment:
This pattern seems a little loose because it matches "TEST" also which is not IP address.
We might want to have more strict pattern as a test.
BTW, I guess we will re-use this pattern for IPv6 support in near future. If so, it is nice to have this pattern in `test/e2e/framework`.

Reply:
@oomichi I added a new regex and made it available as a constant in the framework pkg.
 I couldn´t find a simpler regex, that wasn´t too loose or that covered all the IPv6 addresses